packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -452,6 +495,12 @@ export default {
             if (!this.isFormValid()) {
                 const validationState = validations.getFormValidationState(this.$v);
                 this.$emit(EventNames.CheckoutValidationError, validationState);
+
+                this.$logger.logWarn(
+                    'Checkout Validation Error',
+                    this.$store,
+                    validationState
+                );
```

Comment:
Do we want to log this? (a question, not a statement 😃)

Reply:
The validation error? I think so. It would be useful to see if people are regularly having errors on a specific field.
I know registration has it at the moment. So I was mostly doing it to match. 
_(I'm aware this reasoning has been my downfall for other checkout decisions I've made in the past though 😂 )_